messaging/src/main/java/org/frankframework/jms/JmsMessagingSource.java

```diff
@@ -69,6 +71,11 @@ public Destination lookupDestination(String destinationName) throws JmsException
 		return dest;
 	}
 
+	/**
+	 * Make it so you can fetch these dynamically through an {@link IObjectLocator}.
+	 * In our tests we always 'assume' the client can create destinations.
+	 * This may not always be the case.
+	 */
```

Comment:
Created #8851 though I'm unsure if this has any priority or not.
It certainly makes cleaning this code a lot more difficult...